estuary/utils/story.py

```diff
@@ -185,6 +188,13 @@ def get_wait_times(self, results):
 
             next_artifact_start_time = datetime.strptime(next_artifact.timeline_timestamp,
                                                          '%Y-%m-%dT%H:%M:%SZ')
+
+            if next_artifact.__label__ in ['Advisory', 'ContainerAdvisory'] and \
+                    artifact.__label__ in ['KojiBuild', 'ContainerKojiBuild', 'ModuleKojiBuild']:
```

Comment:
I don't think this "and" is necessary. An advisory is always on its own or after these labels.